extensions/amp-audio/1.0/component.js

```diff
@@ -0,0 +1,195 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as Preact from '#preact';
+import {listen} from '../../../src/event-helper';
+import {setMediaSession} from '../../../src/mediasession-helper';
+import {forwardRef} from '#preact/compat';
+import {triggerAnalyticsEvent} from '../../../src/analytics';
+
+const {useCallback, useEffect, useImperativeHandle, useMemo, useRef} = Preact;
+
+/**
+ * @param {!AudioDef.Props} props
+ * @param {{current: (!AudioDef.AudioApi|null)}} ref
+ * @return {PreactDef.Renderable}
+ */
+export function AudioWithRef(props, ref) {
+  const audioRef = useRef();
```

Comment:
I believe this should be initialized with `null` ⬇️ 
```suggestion
  const audioRef = useRef(null);
```